woocommerce/woocommerce-admin
# update default capability for analytics report to view_woocommerce_reports

Fixes #1968

This PR updates the Analytics menu and submenu items capability permission to `view_woocommerce_reports`. Submenu item capability permission can be changed using the `woocommerce_admin_report_menu_items` filter.

### Detailed test instructions:

- Log in as a shop manager
- View various analytics screens
- Add a filter to `woocommerce_admin_report_menu_items` and add a `capability` => `manage_options` element to one of the menu items
- refresh the screen with to see that screen not included in the menu
